publication-commons/src/test/java/no/unit/nva/publication/service/impl/ResourceServiceTest.java

```diff
@@ -1664,6 +1665,22 @@ void shouldSetFileImportedEventWhenFileIsBeingImported() throws BadRequestExcept
         assertInstanceOf(FileImportedEvent.class, updatedFileEntry.getFileEvent());
     }
 
+    @Test
+    void updateResourceMethodShouldRefreshTickets() throws ApiGatewayException {
+        var publication = randomPublication().copy().withAssociatedArtifacts(List.of()).build();
+        var userInstance = UserInstance.fromPublication(publication);
+        publication = Resource.fromPublication(publication).persistNew(resourceService, userInstance);
+
+        var ticket = createDoiRequest(publication);
+
+        Resource.fromPublication(publication).update(resourceService, userInstance);
+
+        var refreshedTicket = ticket.fetch(ticketService);
+
+        assertNotEquals(ticket.getModifiedDate(), refreshedTicket.getModifiedDate());
```

Comment:
Are we updating anything else & can it be tested?

Reply:
only modified date